.gitattributes

```diff
@@ -0,0 +1,3 @@
+/site/** -linguist-detectable
+/test/** -linguist-detectable
```

Comment:
I wonder if we can do something like `/test/**/samples/**` here? I kind of think it makes sense to consider the test code itself, but the samples might make more sense to exclude. That should alter the final results much from what you have now since most of the code is sample code